test-loop-tests/src/utils/peer_manager_actor.rs

```diff
@@ -242,40 +307,123 @@ impl Handler<PeerManagerMessageRequest> for TestLoopPeerManagerActor {
     }
 }
 
+#[derive(Debug, Clone)]
```

Comment:
Could you please add a comment explaining what does this enum do and what are the effects of the different options?